triggers/runner/src/main/resources/logback.xml

```diff
@@ -1,9 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder>
-            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg %replace(, context: %marker){', context: $', ''} %n</pattern>
-        </encoder>
+    <appender name="STDOUT" class="net.logstash.logback.appender.LoggingEventAsyncDisruptorAppender">
+        <appender-ref ref="console" />
```

Comment:
Apparently, we should no longer nest appenders - see https://jira.qos.ch/browse/LOGBACK-1674